apps/nouns-camp/src/components/candidate-edit-dialog.js

```diff
@@ -156,6 +181,31 @@ const CandidateEditDialog = ({ candidateId, isOpen, close: closeDialog }) => {
     }
   };
 
+  React.useEffect(() => {
+    // ignore creating drafts when present, dismissing changes or submitting candie
+    if (draft != null || hasPendingDismiss || hasPendingSubmit) return;
+
+    createDraft({
```

Comment:
not a huge fan of this, but the alternative of creating an empty draft, and having 3 or 4 sets to initialize it also feels wrong. thoughts?

Reply:
Think we could just get by with a bare useCachedState() and completely bypass the drafts file for edits? That way we wouldn’t have to care about creating them, just the clear call on submit/dismiss. I had a pass at merging this with the drafts hook on `main` (now with topics) but I couldn’t get it to a very satisfying state. I’m not very happy with how it works as is, so feel reluctant to add more to it.